amp/repository/aim/view/teamAuditList.jsp

```diff
@@ -20,6 +20,12 @@ function showUser(email){
 	}
 }
 
+function compare(activityOneId) {

```

Comment:
this function does the same as the one we have in auditlogger. it can be extracted to a .js file and included in both places so we can reuse and avoid having duplicate code